### ISSUE TITLE
TorXakis doesn't build anymore on windows

### ISSUE THREAD

**Issue by pjljvandelaar** [TorXakis/TorXakis#228]
TorXakis has a dependency on text-show-3.6.
On Windows, we are not allowed to use integer-gmp (see https://github.com/commercialhaskell/stack/issues/399).
Since text-show-3.6 uses integer-gmp, we currently have an issue
```
C:\TorXakis>stack build --fast

Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for text-show-3.6:
    integer-gmp must match -any, but the stack configuration has no specified version (latest applicable is 1.0.0.1)
needed due to defs-0.2.1.0 -> text-show-3.6

Recommended action: try adding the following to your extra-deps in C:\TorXakis\stack.yaml:
- integer-gmp-1.0.0.1
```
On https://hackage.haskell.org/package/text-show I can't find a flag to use integer-simple.
I have raised an issue on test-show (https://github.com/RyanGlScott/text-show/issues/31).
But maybe we need a faster action to resolve this issue!

**Comment by pjljvandelaar** [comment]
This bug makes issue #195 a high-priority: the windows CI could have prevented this bug from being incorporated in the development branch!

**Comment by pjljvandelaar** [comment]
Solved by #230
